frappe/contacts/address_and_contact.py

```diff
@@ -165,6 +165,10 @@ def filter_dynamic_link_doctypes(doctype, txt, searchfield, start, page_len, fil
 	_doctypes = tuple([d for d in _doctypes if re.search(txt+".*", _(d[0]), re.IGNORECASE)])
 
 	all_doctypes = [d[0] for d in doctypes + _doctypes]
+	# Company does not have a contact_html field.
+	if filters['fieldname'] == 'contact_html':
+		all_doctypes.append('Company')
```

Comment:
@noahjacob 
this part of the code does not know what company is
Not ideal to hardcode a doctype name here
Can we understand the actual reason behind those filters and maybe edit those to make them more inclusive?